wd_helper.au3

```diff
@@ -1503,32 +1503,35 @@ EndFunc   ;==>_WD_GetBrowserVersion
 ;                  Failure - "" and sets @error to one of the following values:
 ;                  - $_WD_ERROR_InvalidValue
 ;                  - $_WD_ERROR_NotFound
-;; Author ........: Danp2
-; Modified ......: Mlipok
+; Author ........: Danp2
+; Modified ......: mLipok
 ; Remarks .......:
 ; Related .......:
 ; Link ..........:
 ; Example .......: No
 ; ===============================================================================================================================
 Func _WD_GetBrowserPath($sBrowser)
 	Local Const $sFuncName = "_WD_GetBrowserPath"
+	Local Const $sRegKeyCommon = '\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\'
 	Local $iErr = $_WD_ERROR_Success
 	Local $sEXE, $sPath = ""
-	Local $cRegKey = '\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\'
 
 	Local $iIndex = _ArraySearch($_WD_SupportedBrowsers, $sBrowser, Default, Default, Default, Default, Default, $_WD_BROWSER_Name)
 	If @error Then
 		$iErr = $_WD_ERROR_InvalidValue
 	Else
 		$sEXE = $_WD_SupportedBrowsers[$iIndex][$_WD_BROWSER_ExeName]
-		$sPath = RegRead("HKLM" & $cRegKey & $sEXE, "")
 
-		If @error Then $sPath = RegRead("HKCU" & $cRegKey & $sEXE, "")
+		; check HKLM or in case of error HKCU
+		$sPath = RegRead("HKLM" & $sRegKeyCommon & $sEXE, "")
+		If @error Then $sPath = RegRead("HKCU" & $sRegKeyCommon & $sEXE, "")
 
+		; if both checks not found key in the registry
```

Comment:
This isn't very clear. I would recommend something like 

`; Generate Not Found error if neither key is found`

Reply:
Take a look: [comments fixed](https://github.com/Danp2/au3WebDriver/pull/223/commits/b479f1302bd62a8ae0cad505a5a10ac714228ebb)